next-app/src/lib/server/sf.ts

```diff
@@ -1,18 +1,27 @@
 import { error } from '@sveltejs/kit'
+import type { HttpMethod } from '@sveltejs/kit/types/private'
 
-/**
- *
- * @typedef RPC
- * @type {object}
- * @property {string} name Name of the remote procedure to call
- * @property {string[]} [args] Arguments to pass to the remote procedure
- * @property {string} [cookie] https://developer.mozilla.org/en-US/docs/Web/API/Document/cookie
- *
- * @param { RPC } rpc
- */
-export async function sf(rpc) {
-	const { name, args = [], cookie } = rpc
+export type Rpc = {
+	name: string,
+	args?: string[] | object[],
```

Comment:
Did you maybe mean this: an array with both strings as well as objects. As apposed to an array of either strings or objects.
```suggestion
	args?: (string | object)[],
```

Reply:
I've only seen, so far, args as an array of strings or objects.  I believe I represented that correct, would you agree?

`string[]`:
![image](https://user-images.githubusercontent.com/4412848/211360657-b374a98e-7079-437c-a542-8a7a623c0c74.png)

![image](https://user-images.githubusercontent.com/4412848/211360750-80acdd01-ae8b-42a0-898d-11215c7fe9cd.png)

![image](https://user-images.githubusercontent.com/4412848/211360909-751a05f1-f206-4f54-8c90-a95494c6d50c.png)

`object[]`:
![image](https://user-images.githubusercontent.com/4412848/211361134-5ae564e7-1650-4d20-be51-fab4a3ce7c60.png)